### DISCUSSION TITLE
Tailwind CSS installation question

### DISCUSSION THREAD

**Discussion by ShueiYang** [vercel/next.js#52689] [App Router] [answered]
Hello, 
Usually I install tailwind manually in Nextjs and until now I install as DEV dependency, however since Nextjs now also suggest to install and preconfig it together to gain some times, but i notice the installation is on dependency and not in dev, so is there a particular reason?  since Tailwind docs always recommand to install on the dev, since we only use it on dev mode ?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

You can read a bit more here:

- https://github.com/vercel/next.js/issues/51161#issuecomment-1588848409
- https://github.com/vercel/next.js/issues/51161#issuecomment-1590479944
